app/src/main/java/com/android/brewr/ui/userProfile/Information.kt

```diff
@@ -0,0 +1,66 @@
+package com.android.brewr.ui.userProfile
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.outlined.ArrowBack
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.android.brewr.ui.navigation.NavigationActions
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun InformationAboutUsScreen(navigationActions: NavigationActions) {
+
+  Scaffold(
+      modifier = Modifier.fillMaxSize().testTag("UserProfileScreen"),
+      topBar = {
+        TopAppBar(
+            title = { Text("About BrewR") },
+            modifier = Modifier.testTag("topBar"),
+            navigationIcon = {
+              IconButton(
+                  onClick = { navigationActions.goBack() }, Modifier.testTag("goBackButton")) {
+                    Icon(
+                        imageVector = Icons.AutoMirrored.Outlined.ArrowBack,
+                        contentDescription = "Back")
+                  }
+            })
+      },
+      content = { paddingValues ->
+        Column(
+            modifier = Modifier.fillMaxSize().padding(40.dp).padding(paddingValues),
+            verticalArrangement = Arrangement.spacedBy(16.dp)) {
+              Text(
+                  text =
+                      "BrewR is a coffee exploration app created by a team of five EPFL students as part of the Software Enterprise (SwEnt) course. Our goal is to enhance the coffee experience for enthusiasts by combining technology with a love for coffee culture.",
+                  modifier = Modifier.testTag("text1"))
+
+              Text(
+                  text =
+                      "With BrewR, you can:\n" +
```

Comment:
Very well done for the description. However, I would like you to add more "user stories" descriptions that you forgot to mention it, such as : "We can sign in with google" etc.